### ISSUE TITLE
[Bug]: Crashing Homebridge at 1am every day

### ISSUE THREAD

**Issue by mpjalexander** [homebridge-eufy-security/plugin#84]
### What happened?

Since updating to v2.0.0, at 1am every day Homebridge crashes and restarts. This is shown in the log at 1am for the last couple of days:
```
[17/06/2022, 01:00:00] Error: ENOENT: no such file or directory, rename '/var/lib/homebridge/eufysecurity/log-lib.log' -> '/var/lib/homebridge/eufysecurity/log-lib.log.0'
[17/06/2022, 01:00:00] Got SIGTERM, shutting down Homebridge...
```

```
[18/06/2022, 01:00:00] Error: ENOENT: no such file or directory, rename '/var/lib/homebridge/eufysecurity/log-lib.log.1' -> '/var/lib/homebridge/eufysecurity/log-lib.log.2'
[18/06/2022, 01:00:00] Got SIGTERM, shutting down Homebridge...
```

The permissions on the `eufysecurity` directory are:
```
drwxr-xr-x 2 pi pi 4.0K Jun 18 01:00  eufysecurity/
```

The current contents of the directory:
```
-rw-r--r-- 1 pi pi 3.2K Jun 18 01:01 log-lib.log
-rw-r--r-- 1 pi pi 1.7K Jun 17 12:03 log-lib.log.0
-rw-r--r-- 1 pi pi 3.5K Jun 16 18:54 log-lib.log.2
-rw-r--r-- 1 pi pi 1.9K Jun 18 01:01 persistent.json
-rw-r--r-- 1 pi pi  802 Jun  3 09:57 stations.json
```

Is there some issue with the log rotation logic? I definitely haven’t removed the `log-lib.log.1` file that generated the error.

### Device Type

Indoor Camera (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[18/06/2022, 01:00:00] Error: ENOENT: no such file or directory, rename '/var/lib/homebridge/eufysecurity/log-lib.log.1' -> '/var/lib/homebridge/eufysecurity/log-lib.log.2'
[18/06/2022, 01:00:00] Got SIGTERM, shutting down Homebridge...
```

**Comment by thieren** [comment]
Hi @mpjalexander 

this is a duplicate of #56 
This is fixed in the beta and will be in the next release.

Thank you!

**Comment by mpjalexander** [comment]
Hi @thieren. I see in #56 that this issue was apparently fixed in 2.0.1-beta.4. I'm running 2.0.1, but still have the problem with the plug-in crashing Homebridge at 1am every day; the same error message I originally reported. Any ideas?

**Comment by thieren** [comment]
We had to correct the versioning for the plugin some time ago due to a quick bugfix in the master.
The 2.0.1 beta you mentioned was uplifted and is now the 2.1.0-rc 
Please try again with the release candidate.

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
